### PR TITLE
add caption to duvivier slide

### PR DESCRIPTION
**Pull request by duviviereib**
Please check

### CODE REVIEW

**Review by florin-d — CHANGES_REQUESTED** [review]
Come'on: blabla is not allowed in code, be more precise!

**Comment by duviviereib** [comment]
It is the name of the cat: blabla

**Comment by gdebrito** [comment]
great job 🔪

File: _posts/0000-01-12-duviviereib.md [hunk 1]
```diff
@@ -7,3 +7,4 @@ title: "Welcome to our slide deck!"
 {: .center}
 
 Caption here
+blabla
```
Comment (jacquemf) [inline _posts/0000-01-12-duviviereib.md:10]: Arnaud, à revoir !
F.

**Review by gdebrito — APPROVED** [review]

**Review by jacquemf — APPROVED** [review]

**Review by denisnoel — APPROVED** [review]
joli nom

**Review by arnaudolig — APPROVED** [review]
Quelle maitrise!

**Review by eib-fabianburtard — APPROVED** [review]
Amazing

**Review by gdebrito — APPROVED** [review]

**Review by florin-d — APPROVED** [review]
Bla bla bla

**Comment by duviviereib** [comment]
Finally

**Pull request merged**